widgets-samples/cc/samples-cc-react-app/src/App.tsx

```diff
@@ -328,6 +329,12 @@ const handleSaveEnd = (isComplete: boolean) => {
 
   const onStateChange = (status) => {
     console.log('onStateChange invoked', status);
+    if(!status || !status.name)return;
+    if(status.name !== 'RONA'){
+      setShowRejectedPopup(false);
+    }else{
+      setShowRejectedPopup(true);
+    }
```

Comment:
It looks like this code doesn't follow the ESLint rules setup by Shreyas. 

Could you please install [ESLint VS Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension and try to save this file again? That should correct the formatting issues in this file